### DISCUSSION TITLE
Image Upload Via Api

### DISCUSSION THREAD

**Discussion by FashMuyhee** [adonisjs/core#2463] [Help]
Hello, I am working on  an API that requires users to upload image for their profile picture, it works well when I test with either postman or isomnia but when test with axois via react-native i keep getting network error.
![code-snapshot-6](https://user-images.githubusercontent.com/34525848/114019445-44c57280-9823-11eb-8450-7290bbaeb195.png)
_my  react-native code that make use of the image upload_
**N.B** : the image is passed currently and when console, the image uri, filetype, and size are consoled
![code-snapshot-5](https://user-images.githubusercontent.com/34525848/114019451-468f3600-9823-11eb-8a9a-c1062e4ccf86.png)

_adonis code for the image upload_

**Comment by thetutlage** [comment]
Then it seems more like a Question for react native or axios than AdonisJS? AdonisJS needs multipart requests (the standard for uploading files), you need to find out how to do that in Axios
